Gemfile.lock

```diff
@@ -246,15 +246,12 @@ GEM
       nokogiri (>= 1.6)
     jbuilder (2.10.0)
       activesupport (>= 5.0.0)
-    jquery-datetimepicker-rails (2.4.1.0)
```

Comment:
Was this removed earlier?

Reply:
That is just an artefact of me playing around with that gem.